playbooks/roles/helm/tasks/setup_helm.yml

```diff
@@ -0,0 +1,41 @@
+---
+# git clone temporary repository while the PR continue in review:
+# https://github.com/minishift/minishift-addons/pull/92
+- name: "checkout the pipeline @ {{ pipeline_repo }} to {{ pipeline_dir }}"
+  git:
+    repo: "https://github.com/firemanxbr/minishift-addons.git"
```

Comment:
PR https://github.com/minishift/minishift-addons/pull/92 seems to be merged. So it would be good to use official minishift repository for addons.

Reply:
I'll change it in the next commit. We are using it because my PR was in review over there:  https://github.com/minishift/minishift-addons/pull/92 
Now it's approved.